pepper-apis/src/main/java/org/broadinstitute/ddp/util/UserAdminCLI.java

```diff
@@ -42,14 +42,16 @@
     public static final String EMAIL_OPTION = "e";
     public static final String PASSWORD_OPTION = "p";
     public static final String CLIENT_OPTION = "c";
+    public static final String TENANT_OPTION = "t";
 
     public static void main(String[] args) {
         Options options = new Options();
 
         options.addRequiredOption(STUDY_OPTION, "study", true, "study guid (note caps sensitivity");
         options.addRequiredOption(EMAIL_OPTION, "email", true, "email for the account");
         options.addRequiredOption(PASSWORD_OPTION, "password", true, "password for the account");
-        options.addRequiredOption(CLIENT_OPTION, "client", true, "auth client id");
+        options.addRequiredOption(CLIENT_OPTION, "client", true, "auth0 client id");
+        options.addRequiredOption(TENANT_OPTION, "tenant", true, "auth0 tenant id");
```

Comment:
Should we take a `auth0Domain` instead?

Reply:
I get a bit confused by `domain`, especially with [auth0 custom domains](https://auth0.com/docs/custom-domains).  Tenant is less confusing to me.

Reply:
Again, might be better to take the tenant/domain string instead of numeric id...

Reply:
@yufengwng, not quite sure what the tenant string is. Could you please give an example?

Reply:
Yup, something like `https://ddp-dev.auth0.com/`.